test/elixir/test/basics_test.exs

```diff
@@ -58,6 +58,64 @@ defmodule BasicsTest do
     assert context[:db_name] in Couch.get("/_all_dbs").body, "Db name in _all_dbs"
   end
 
+  @tag :with_db
+  test "Database name with '+' should encode to '+'", _context do
+    set_config({"chttpd", "encode_plus", "false"})
+
+    random_number = :rand.uniform(16_000_000)
+    db_name = "random+test+db+#{random_number}"
+    resp = Couch.put("/#{db_name}")
+
+    assert resp.status_code == 201
+    assert resp.body["ok"] == true
+
+    resp = Couch.get("/#{db_name}")
+
+    assert resp.status_code == 200
+    assert resp.body["db_name"] == db_name
+  end
+
+  @tag :with_db
```

Comment:
Again with `@tag :with_db`  you are creating a database which you don't use in test.

Reply:
See above.